helm_deploy/values-dev.yaml

```diff
@@ -30,7 +30,7 @@ env:
   OUTBOUND_DELETION_SQS_PROVIDER: aws
   INBOUND_REFERRAL_SQS_PROVIDER: aws
 
-  SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK-SET-URI: "https://gateway.t3.nomis-api.hmpps.dsd.io/auth/.well-known/jwks.json"
+  SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI: "https://gateway.t3.nomis-api.hmpps.dsd.io/auth/.well-known/jwks.json"
```

Comment:
Have you been able to test this change yet Jon?  I've got the same var but haven't merged yet so haven't tested the deployment...